vaadin-spreadsheet-charts/pom.xml

```diff
@@ -132,6 +134,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>license-maven-plugin</artifactId>
+                <version>2.4.0</version>
                 <configuration>
                     <licenseName>cval3</licenseName>
```

Comment:
VCL

Reply:
That's going to be a different PR, loads of changes needed for that so easier to review them separately.